.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,17 +1,22 @@
-<!-- Please take a look at our Contributing documentation before submitting a Pull Request!
-https://rook.io/docs/rook/latest/Contributing/development-flow/
+<!-- Thank you for contributing to Rook! -->
 
-Thank you for contributing to Rook! -->
+<!-- STEPS TO FOLLOW:
+  1. Add a description of the changes (frequently the same as the commit description)
+  2. Enter the issue number next to "Resolves #" below (if there is no tracking issue resolved, **remove that section**)
+  3. Review our Contributing documentation at https://rook.io/docs/rook/latest/Contributing/development-flow/
+  4. Follow the steps in the checklist below, starting with the **Commit Message Formatting**.
+-->
 
-**Description of your changes:**
-
-**Which issue is resolved by this Pull Request:**
+<!-- Uncomment this section with the issue number if an issue is being resolved
+**Issue resolved by this Pull Request:**
 Resolves #
+--->
 
 **Checklist:**
 
 - [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure).
 - [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
-- [ ] CI tests has been updated, if necessary.
+- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
```

Comment:
Is this valid? Are we going to update Rook pending notes when we have a PR here?

Reply:
well, we do mention it in release notes but not sure we need it here, I just copied it from rook/rook. @travisn thoughts?

Reply:
We don't need the checklist items to match from the rook repo where they don't apply. Let's remove the note about pending release notes.

Reply:
removed